bedrock/firefox/forms.py

```diff
@@ -10,8 +10,6 @@
 
 from lib.l10n_utils.dotlang import _lazy as _
 
-LANG_FILES = ['firefox/whatsnew-fx37']
-
```

Comment:
I'm not completely sure that message was ever translated (we have about U.S. phones in sendto.lang).